addons/sys_core/XEH_postInit.sqf

```diff
@@ -42,6 +42,8 @@ DFUNC(gen) = {
 
 ["ACRE2", "HeadSet", (localize LSTRING(HeadSet)), "", { [] call FUNC(toggleHeadset) }, [208, [true, true, false]]] call cba_fnc_addKeybind;
 
+["ACRE2", "AntennaDirToggle", (localize LSTRING(AntennaDirToggle)), "", { [] call EFUNC(sys_components,toggleAntennaDir) }, [200, [false, true, true]]] call cba_fnc_addKeybind;
```

Comment:
Let's prefix `"AnennaDirToggle"`, just in case. Old ones should be too, but too late for that.
Also `CBA_fnc_addKeybind`.